tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -120,19 +120,19 @@ def _test_cluster_limits(remote_command_executor, max_queue_size, region, asg_na
     slurm_commands = SlurmCommands(remote_command_executor)
     result = slurm_commands.submit_command("sleep 1000", nodes=max_queue_size + 1)
     max_nodes_job_id = slurm_commands.assert_job_submitted(result.stdout)
-    result = remote_command_executor.run_remote_command("sbatch -N 1 --wrap='sleep 1' --cpus-per-task 5")
-    max_cpu_job_id = slurm_commands.assert_job_submitted(result.stdout)
+    try:
+        result = remote_command_executor.run_remote_command("sbatch -N 1 --wrap='sleep 1' --cpus-per-task 5")
```

Comment:
you can call it with `raise_on_error=False` and assert the result without the try-except